partner_feeds/static/partner_feeds/css/select-feed.css

```diff
@@ -174,7 +174,7 @@
     height: 20px;
 
     cursor: pointer;
-    background-image: url(/static/grappelli/images/icons-s96d5c23000.png);
+    background-image: url(../../grappelli/images/icons-s0e29227ce9.png);
```

Comment:
Not sure why I thought it a good idea to reference grappelli's spritesheet. Sorry dude.